linux_os/guide/system/software/gnome/gnome_system_settings/dconf_gnome_disable_ctrlaltdel_reboot/ansible/shared.yml

```diff
@@ -8,7 +8,7 @@
     dest: /etc/dconf/db/local.d/00-security-settings
     section: org/gnome/settings-daemon/plugins/media-keys
     option: logout
-    value: string ''
+    value: "''"
     create: yes
```

Comment:
Please add
no_extra_spaces: yes
because without this, the remediation creates
logout = ''
and this fails the OVAL.
Or, alternatively, modify the OVAL to allow these extra spaces.

Reply:
Thanks for investigating that... I was about to check this. I will decide which approach to take.